### ISSUE TITLE
[7.4.1] Unicode problem in torrent NAME

### ISSUE THREAD

**Issue by Dmole** [Tribler/tribler#5112]
"Логан" (Russian for Logan) shows up as 
![tribler](https://user-images.githubusercontent.com/1793802/74039670-3fa33400-4990-11ea-8144-ce30ac0e5433.png)

**Comment by ichorid** [comment]
The question is *where* is it shown incorrectly? In the channels? In the downloads list? In the downloads files list? Where exactly?

**Comment by Dmole** [comment]
downloads

**Comment by ichorid** [comment]
Well, this seems to be fixed in 7.5 branch. We'll see if we're going to backport the fix to 7.4.x

**Comment by ichorid** [comment]
Hmm... Can't reproduce it with `release-7.4.1` on both Linux and Windows. Used some torrents with Cyrillic names in both files and the torrent name itself. Looks fine everywhere.

**Comment by Dmole** [comment]
@ichorid please try this;

```
echo test > test.txt
mktorrent -a 127.0.0.1 -n Логан_breaks_here -o test.torrent test.txt
mv test.torrent ~/Downloads/Tribler/Watch/
```

**Comment by ichorid** [comment]
Ok, it worked on both 7.4 an `devel`. I've caught the bug. `mktorrent` does not add the `encoding` key to the torrent, so Tribler uses default encoding. However, the string representing the encoding itself is now `bytes` in Py3, and that produced exactly the same error code as the natural `.decode` error supposed to! Most torrent files contain the proper encoding string, so this was a corner case

 Thanks for the report!

**Comment by ichorid** [comment]
solved by #5120
